doc/specs/stdlib_io.md

```diff
@@ -131,3 +131,126 @@ program demo_savetxt
     call savetxt('example.dat', x) 
 end program demo_savetxt
 ```
+
+## `disp` - display the value of the variable
+
+### Status
+
+Experimental
+
+### Class
+
+Impure subroutine.
+
+### Description
+
+Outputs a `logical/integer/real/complex/character/string_type` scalar,  
+or `logical/integer/real/complex/string_type` and rank-1/rank-2 array to the screen or a file `unit`.
+
+### Syntax
+
+`call [[stdlib_io(module):disp(interface)]]( [x, header, unit, brief, format, width, sep] )`  
+
+### Arguments
+
+- `x`: Shall be a `logical/integer/real/complex/character(len=*)/string_type` scalar or `logical/integer/real/complex/string_type` and rank-1/rank-2 array.
+This argument is `intent(in)` and `optional`.
+
+- `header`: Shall be a `character(len=*)` scalar. 
+This argument is `intent(in)` and `optional`.
+
+- `unit`: Shall be an `integer` scalar, linked to an IO stream.
+This argument is `intent(in)` and `optional`.<br>
+The default value is `output_unit` from `iso_fortran_env` module.
+
+- `brief`: Shall be a `logical` scalar, controls an abridged version of the `x` array to be outputted.
+This argument is `intent(in)` and `optional`.<br>
+The default value is `.false.`
+
+- `format`: Shall be a `character(len=*)` scalar.
+This argument is `intent(in)` and `optional`.<br>
+The default value is `g0.4`.
+
+- `width`: Shall be an `integer` scalar, controls the outputted maximum width (`>=80`).
+This argument is `intent(in)` and `optional`.<br>
+The default value is `80`.
+
+- `sep`: Shall be a `character(len=*)` scalar, separator.
+This argument is `intent(in)` and `optional`.<br>
+The default value is "&ensp;&ensp;", two spaces.
+
+### Output
+
+The result is to print `header` and `x` on the screen (or another output `unit/file`) in this order.<br>
+If `disp` is not passed any arguments, a blank line will be printed.
+
+### Example
+
+```fortran
```

Comment:
I suggest to reduce the example. One or two `disp` should be enough for the reader to understand its aim.